homework/index.js

```diff
@@ -30,17 +15,114 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
-      } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
-      }
+  // Function: display repository information ( left side )
+  function showRepoBasicInfo(container, repository) {
+    const repInfoTable = createAndAppend('table', container);
+    const tr1 = createAndAppend('tr', repInfoTable);
+
+    // raw 1
+    createAndAppend('td', tr1, { class: 'heading', text: 'repository:' });
+    createAndAppend('a', tr1, {
```

Comment:
an `a` tag is not a valid child element for the `tr` so, it may cuase some sort of bugs into your code, Imagine the following:

you have an unorderedlist element which holds a list element, if you changed the `ul` to be an `ol` the list element will be changed to number or letters instead of bullets, but if you have an `h1` inside where the bullets will go?